modules/invibesBidAdapter.md

```diff
@@ -20,7 +20,8 @@ Connect to Invibes for bids.
                    {
                        bidder: 'invibes',
                        params: {
-                           placementId: '12345'
+                           placementId: '12345',
+						   networkId: 1001
```

Comment:
should this be `domainId`? I don't see `networkId` listed anywhere else in your change set

also, can you fix the spacing please

Reply:
yes, it should have been domainId !